css/at-rules/media.json

```
@@ -1293,7 +1293,7 @@
                 "version_added": "67"
               },
               "firefox_android": {
-                "version_added": false
+                "version_added": "79"
```

OK, digging in again. It does appear that `prefers-color-scheme` was supported from 67. I trust the test ahead of some of the Bugzilla bugs (though even some of the bugs seem to indicate that [it was supported](https://bugzilla.mozilla.org/show_bug.cgi?id=1557263#c3)).

```suggestion
                "version_added": "67"
```